bears/general/InvalidLinkBear.py

```diff
@@ -108,13 +115,22 @@ def run(self, filename, file,
         `do_not_ever_open = 'https://api.acme.inc/delete-all-data'` wiping out
         all your data.
 
-        :param timeout:          Request timeout period.
+        :param network_timeout:       A dict containing the url and the timeout
         :param link_ignore_regex:     A regex for urls to ignore.
         :param link_ignore_list: Comma separated url globs to ignore
         :param follow_redirects: Set to true to autocorrect redirects.
         """
+
+        host_dict = dict()
+        for url in network_timeout:
+            host = urlparse(url).netloc
+            timeout = network_timeout[url]
+            host_dict[host] = timeout
```

Comment:
oh and you could also remove the stmt below when assigning to `network_timeout` directly^^